src/python-common/ceph/deployment/service_spec.py

```diff
@@ -1503,8 +1505,19 @@ def __init__(self,
         self.spdk_ping_interval_in_seconds = spdk_ping_interval_in_seconds
         #: ``ping_spdk_under_lock`` whether or not we should perform SPDK ping under the RPC lock
         self.ping_spdk_under_lock = ping_spdk_under_lock
+        #  spdk to ceph connection mapping
+        #: see https://github.com/ceph/ceph-nvmeof?tab=readme-ov-file#mapping-spdk-bdevs-into-a-ceph-rados-cluster-context
         #: ``bdevs_per_cluster`` number of bdevs per cluster
         self.bdevs_per_cluster = bdevs_per_cluster
+        #: ``flat_bdevs_per_cluster`` number of bdevs per cluster, ignore ANA group
+        self.flat_bdevs_per_cluster = flat_bdevs_per_cluster
+        #: ``cluster_connections`` number of ceph cluster connections
+        self.cluster_connections = cluster_connections
+        # set default only if all spdk alloc stratgies are None (no parameters explicitly defined)
+        if not any([self.bdevs_per_cluster is not None,
+                   self.flat_bdevs_per_cluster is not None,
+                   self.cluster_connections is not None]):
+            self.cluster_connections = 32
```

Comment:
Nitpick, I think this works, but just to avoid the double negative
```suggestion
        if all([self.bdevs_per_cluster is None,
                   self.flat_bdevs_per_cluster is None,
                   self.cluster_connections is None]):
            self.cluster_connections = 32
```